controllers/customer.js

```diff
@@ -136,13 +95,11 @@ exports.create = (req, res) => {
 //Get single customer
 exports.findOne = (req, res) => {
 
-  validateId(req.params.id, res);
-
-  Customer.findOne({ id: req.params.id })
+  Customer.findOne({ _id: req.params.id })
     .then((data) => {
       if (!data) {
         return res.status(404).send({
-          message: `Customer with id ${req.params.id} was not found`,
+          message: `Customer with id ${req.params.id} does not found`,
```

Comment:
estaba bien escrito antes jajajaja
 `Customer with id ${req.params.id} was not found`, --> El cliente con el id no fue encontrado. --> bien.
`Customer with id ${req.params.id} does not found`, --> El cliente con el id no es encontrado --> mal.